lint-configs/.clang-format

```diff
@@ -20,16 +20,15 @@ AllowAllArgumentsOnNextLine: false
 AllowAllParametersOfDeclarationOnNextLine: false
 AllowBreakBeforeNoexceptSpecifier: "OnlyWithParen"
 AllowShortBlocksOnASingleLine: "Always"
+AllowShortCaseExpressionOnASingleLine: false
```

Comment:
This is an option for Java code (rather than C++), right?

Reply:
true, I will remove it